github/actions_runners.go

```diff
@@ -151,12 +151,18 @@ type Runners struct {
 	Runners    []*Runner `json:"runners"`
 }
 
+// ListRunnersOptions specifies the optional parameters to the ListRunners and ListOrganizationRunners methods.
+type ListRunnersOptions struct {
+	Name string `url:"name,omitempty"`
```

Comment:
```suggestion
	Name *string `url:"name,omitempty"`
```

Reply:
Done 👍 
I hope you don't mind a side question. I looked at other `List*Options`. Some use ptr values, and others do not. What is the rule here?

Reply:
Sure! Mandatory params are passed by value and optional params are passed by reference. So `nil` opts just uses the defaults for that endpoint.